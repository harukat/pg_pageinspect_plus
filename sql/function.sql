SELECT tuple_data_parse('public.t1'::regclass, tuple_data_split('public.t1'::regclass, t_data, t_infomask, t_infomask2, t_bits)) FROM heap_page_items(get_raw_page('public.t1', 0));
SELECT tuple_data_parse('public.t2'::regclass, tuple_data_split('public.t2'::regclass, t_data, t_infomask, t_infomask2, t_bits)) FROM heap_page_items(get_raw_page('public.t2', 0));
