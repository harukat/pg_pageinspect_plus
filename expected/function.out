SELECT tuple_data_parse('public.t1'::regclass, tuple_data_split('public.t1'::regclass, t_data, t_infomask, t_infomask2, t_bits)) FROM heap_page_items(get_raw_page('public.t1', 0));
                     tuple_data_parse                      
-----------------------------------------------------------
 {100,ABCDE,"Sat Aug 01 12:00:00 2020",true,98765432}
 {101,abcde,"Fri Aug 01 12:00:00 2025",false,554433221100}
(2 rows)

SELECT tuple_data_parse('public.t2'::regclass, tuple_data_split('public.t2'::regclass, t_data, t_infomask, t_infomask2, t_bits)) FROM heap_page_items(get_raw_page('public.t2', 0));
                                           tuple_data_parse                                            
-------------------------------------------------------------------------------------------------------
 {xxx,"XXX  ","Sun Aug 02 05:19:00.698 2020 PDT",1.234,9.87654321,"[\"a\",\"b\",\"c\"]","@ 100 hours"}
(1 row)

